### DISCUSSION TITLE
Welcome to canal-2 Discussions!

### DISCUSSION THREAD

**Discussion by haruna-rashid-yakubu** [Worketyamo-Students/canal-2#1] [Announcements]
# Nous discuterons du projet Canal 2 ici

**Comment by Jonnorich99** [comment]
bonjour Mr. j'ai rencontré des difficultés a introduire les balises "main" et "footer", j'ai beau chercher le problème dans la pages css en vain, j'ai du mettre tout le contenu dans "header".
Et aussi, j'aurai savoir quand, comment et pourquoi il faut utiliser "em" "rem" "vh" "%"  "wh" ? bref quand utilise-t-on ces unité?
